components/tyk-dashboard/values.yaml

```diff
@@ -243,6 +243,9 @@ dashboard:
   # that stores this data.
   # It is used to set TYK_DB_ENABLEANALYTICSCACHE
   enableAnalyticsCache: true
+  # This allows the modification of Tyk OAS APIs via the Tyk Classic API endpoints. 
+  # It is used to set TYK_DB_ALLOWUNSAFEOAS
+  allowUnsafeOAS: false  
```

Comment:
I would not add this option in helm charts, as there are plans to deprecate this config from Dashboard in future.
User can still set it if needed using extraenvs